src/screens/Dashboard/ThreadList/ThreadItem/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { FC, memo } from 'react';
 import { StyleSheet, View } from 'react-native';
 import { Text } from '@metacraft/ui';
+import CommentInput from 'components/CommentInput';
 
 import BellIcon from '../../../../components/icons/feather/Bell';
 import PinIcon from '../../../../components/icons/feather/Pin';
```

Comment:
use aliased import, just like `CommentInput` above:
- `import BellIcon from 'components/icons/feather/Bell'`;
- `import PinIcon from 'components/icons/feather/Pin'`;

Reply:
I've fixed it, pls check again